x/concentrated-liquidity/keeper_test.go

```diff
@@ -111,13 +111,21 @@ func (s *KeeperTestSuite) SetupOverlappingRangePositionAcc(poolId uint64, owner
 }
 
 // validatePositionUpdate validates that position with given parameters has expectedRemainingLiquidity left.
-func (s *KeeperTestSuite) validatePositionUpdate(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, lowerTick int64, upperTick int64, joinTime time.Time, freezeDuration time.Duration, expectedRemainingLiquidity sdk.Dec) {
-	newPositionLiquidity, err := s.App.ConcentratedLiquidityKeeper.GetPositionLiquidity(ctx, poolId, owner, lowerTick, upperTick, joinTime, freezeDuration)
+func (s *KeeperTestSuite) validatePositionUpdate(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, lowerTick int64, upperTick int64, joinTime time.Time, freezeDuration time.Duration, positionId uint64, expectedRemainingLiquidity sdk.Dec) {
+	newPositionLiquidity, err := s.App.ConcentratedLiquidityKeeper.GetPositionLiquidity(ctx, poolId, owner, lowerTick, upperTick, joinTime, freezeDuration, positionId)
 	s.Require().NoError(err)
 	s.Require().Equal(expectedRemainingLiquidity.String(), newPositionLiquidity.String())
 	s.Require().True(newPositionLiquidity.GTE(sdk.ZeroDec()))
 }
 
+// validatePositionsUpdate validates that positions with the given parameters have expectedRemainingLiquidity left.
+func (s *KeeperTestSuite) validatePositionsUpdate(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, lowerTick int64, upperTick int64, expectedRemainingLiquidity sdk.Dec) {
+	newPositionsLiquidity, err := s.App.ConcentratedLiquidityKeeper.GetPositionsLiquidity(ctx, poolId, owner, lowerTick, upperTick)
+	s.Require().NoError(err)
+	s.Require().Equal(expectedRemainingLiquidity.String(), newPositionsLiquidity.String())
+	s.Require().True(newPositionsLiquidity.GTE(sdk.ZeroDec()))
+}
```

Comment:
Why do we need this method and not use the `validatePositionUpdate` by providing an id?

Reply:
Removed logic here https://github.com/osmosis-labs/osmosis/pull/4661/commits/c906adae1baa2cf19a090ec602b4e8e61fa6db9d